### DISCUSSION TITLE
Where to post if it's a Sharepoint search issue ?

### DISCUSSION THREAD

**Discussion by ferrarirosso** [microsoft-search/pnp-modern-search#2389] [Q&A] [answered]
Does anyone knows where to post when you have found a Sharepoint search issue/bug ?

**Comment by wobba** [comment, marked as answer]
If it's API related you can try https://github.com/SharePoint/sp-dev-docs/issues or repro it with a classic search web part and log via Microsoft Support.

**Reply by ferrarirosso** [reply]
Thanks @wobba and done [Sharepoint language detection when page publishing is scheduled](https://github.com/SharePoint/sp-dev-docs/issues/8466)

**Reply by wobba** [reply]
This seems more a product issue than API, so I would log via Microsoft Support in the M365 admin center.
